test/setup_test.go

```diff
@@ -199,6 +223,12 @@ func testSetup() {
 		applyAndWaitForApplications(commitID)
 	})
 
+	// This block is for workaround to avoid tls handshake error due to deletion ceph-ssd-block-pool
+	// Please delete this block after Rook 1.4 is applied
```

Comment:
Use "TODO:" for ease of search.
```suggestion
	// TODO: delete this block after Rook 1.4 is applied
```